hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapred/AccumuloMultiTableInputFormat.java

```diff
@@ -1,99 +0,0 @@
-/*
```

Comment:
I do not think this should be removed, along with its functionality. I seem to recall multiple table scanning to be a high-demand feature when this was first added. However, it is redundant with AccumuloInputFormat, since that is a trivial case of this one. I think the AccumuloInputFormat should be modified to support multiple tables, so we can eliminate one of these, but still keep the ability to scan multiple tables.

Reply:
OK I will create a follow on ticket so we don't lose the multi table functionality.  It shouldn't be too bad to add it as an option to InputInfo.

Reply:
Created #749